src/App/Kernel.php

```diff
@@ -77,24 +78,32 @@ public function dependencies(Application $app)
         $app['schematron.controller'] = function (Application $app) {
             return new DefaultController($app['schematron.backend_client']);
         };
+        $app['ping.controller'] = function() {
```

Comment:
Can be replaced with `$app->register(new PingControllerProvider());` after updating `elife/ping`.